_quarto-default.yml

```diff
@@ -15,7 +15,8 @@ book:
   - "modules/1_read_data/read_data.qmd"
   - "modules/2_preprocessing/rna/preprocessing.qmd"
   - "modules/3_normalization/rna/normalization.qmd"
-  - "modules/6_clusterqc/rna/clusterqc.qmd"
+  - "modules/6_clusterqc/clusterqc.qmd"
+  - "modules/8_cluster_composition/cluster_composition.qmd"
```

Comment:
I just would like to emphasize again that the numbers in the module names do not really reflect the order in which the modules are run :-) . 

I can for example leave out clusterqc then this would be module 7.

Reply:
Du bist ueberstimmt. ;-)